fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
```

Comment:
shouldn't this go to the root gitignore

Reply:
it's for fuzz/target. This file is generated by `cargo fuzz`

Reply:
good to know